azure-client-runtime/src/main/java/com/microsoft/azure/v2/annotations/AzureHost.java

```diff
@@ -0,0 +1,66 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.v2.annotations;
```

Comment:
I'm a little concerned about putting version numbers into our packages/folder structure. Why not just create a v2 branch off of master that we do all of our work in until we're ready for v2 to be our main version, at which point we would just merge v2 into master and then delete v2?